addOns/packscanrules/packscanrules.gradle.kts

```diff
@@ -0,0 +1,38 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+description = "All of the add-ons just containing release, beta and alpha status scan rules"
+
+zapAddOn {
+    addOnName.set("Collection: Scripting Pack")
+    addOnStatus.set(AddOnStatus.ALPHA)
+    zapVersion.set("2.11.1")
+
+    manifest {
+        author.set("ZAP Dev Team")
+        url.set("https://www.zaproxy.org/docs/desktop/addons/collection-scan-rules-pack/")
+
+        dependencies {
+            addOns {
+                register("ascanrules")
+                register("ascanrulesAlpha")
+                register("ascanrulesBeta")
+                register("domxss")
+                register("pscanrules")
+                register("pscanrulesAlpha")
+                register("pscanrulesBeta")
```

Comment:
Retire?
JWT?
FileUpload?

Reply:
Added Retire.js
Not so sure about the other 2 - they are more manual and already in the Pantester Pack..